docs-2.0/nebula-explorer/operation-guide/ex-ug-page-overview.md

```diff
@@ -15,7 +15,9 @@ The main page of Explorer is divided into five parts:
 
 ## Tab bar
 
-- Export: Export a CSV or PNG file of the current view.
+- Export: Exports a CSV or PNG file of the current view.
+
```

Comment:
Exports or Export?
Please keep the same style with the following descriptions below.